pkgs/development/python-modules/inflate64/default.nix

```diff
@@ -0,0 +1,47 @@
+{
+  buildPythonPackage,
+  fetchFromGitea,
+  lib,
+  pytestCheckHook,
+  setuptools,
+  setuptools-scm,
+}:
+
+buildPythonPackage rec {
+  pname = "inflate64";
+  version = "1.0.1";
+
```

Comment:
```suggestion
```

Reply:
I'm not convinced these minor stylistic preferences are worth the friction of requesting changes.

Seems to go against the principles laid out in [How to review pull requests](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#how-to-review-pull-requests) and [A reviewer requested a bunch of insubstantial changes on my PR](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#a-reviewer-requested-a-bunch-of-insubstantial-changes-on-my-pr), though maybe my interpretation is off.

Maybe consider marking such minor feedback as a nitpick, or explain why you think it is important? :slightly_smiling_face: